Jenkinsfile

```diff
@@ -330,7 +330,7 @@ pipeline {
                             bat 'hostname'
                             bat '"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Auxiliary\\Build\\vcvarsall.bat" x86_amd64 && cmake -G"Visual Studio 15 2017 Win64" .. -DCMAKE_BUILD_TYPE=RELEASE -DMEZZ_BuildDoxygen=OFF -DMEZZ_CodeCoverage=OFF'
                             bat 'cmake --build .'
-                            bat 'PackageName_Tester xml'
+                            bat 'IOStreams_Tester xml'
```

Comment:
I think I see the pattern now.

Reply:
?

Reply:
Bad attempt a joke, all of the changes in this file are formulaic.